src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -502,16 +506,14 @@ namespace AirLoopHVACDOAS {
                 int CoolingCoilOrder = 0;
                 int FanOrder = 0;
                 for (int CompNum = 1; CompNum <= thisOutsideAirSys.NumComponents; ++CompNum) {
-                    std::string &CompType = thisOutsideAirSys.ComponentType(CompNum);
-                    std::string &CompName = thisOutsideAirSys.ComponentName(CompNum);
+                    std::string const &CompType = thisOutsideAirSys.ComponentType(CompNum);
+                    std::string const &CompName = thisOutsideAirSys.ComponentName(CompNum);
 
                     bool InletNodeErrFlag = false;
                     bool OutletNodeErrFlag = false;
 
                     const std::string typeNameUC = Util::makeUPPER(thisOutsideAirSys.ComponentType(CompNum));
-                    ValidEquipListType foundType = static_cast<ValidEquipListType>(getEnumValue(validEquipNamesUC, typeNameUC));
-
-                    switch (foundType) {
+                    switch (static_cast<ValidEquipListType>(getEnumValue(validEquipNamesUC, typeNameUC))) {
```

Comment:
Smart enough to make this substitution but uses `auto` in range based loops?

Reply:
I think this was me doing stuff manually.